frontend/src/pages/manage-member/ManageMember.jsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState, useEffect } from "react";
+import "./styles.scss";
+
+import {
+  Button,
+  Col,
+  Row,
+  Typography,
+  Table,
+  Space,
+  Select,
+  Input,
+} from "antd";
+import { RiPencilFill } from "react-icons/ri";
+
+import AddMember from "./AddMember";
+import { api, store } from "../../lib";
+import { FaSearch } from "react-icons/fa";
+
+const { Title } = Typography;
+
+const ManageMember = () => {
+  const { isLoggedIn, optionValues } = store.useState((s) => s);
+  const { organisation, member_type, isco_type } = optionValues;
+  const iscoType = isco_type.filter((isco) => isco.name !== "All");
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [search, setSearch] = useState(null);
+  const [isNewMemberVisible, setIsNewMemberVisible] = useState(false);
+  const [selectedMember, setSelectedMember] = useState(null);
+  const [memberValue, setMemberValue] = useState(null);
+  const [memberNameFilter, setMemberNameFilter] = useState(null);
+  const [memberTypeFilter, setMemberTypeFilter] = useState(null);
+  const [iscoValue, setIscoValue] = useState(null);
+  const [iscoFilter, setIscoFilter] = useState(null);
+  const [reload, setReload] = useState(0);
+  const [data, setData] = useState([]);
+
+  const pageSize = 10;
+  const [page, setPage] = useState(1);
+
+  const columns = [
```

Comment:
Add `width` prop to each column:
- Name: 40%
- Member Type: 25%
- ISCO: 25%
- Action: 10%